akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEventParser.scala

```diff
@@ -34,23 +34,35 @@ private object ServerSentEventParser {
 
   private val linePattern = """([^:]+): ?(.*)""".r
 
-  private def parseServerSentEvent(lines: Vector[String]): ServerSentEvent = {
-    val values = lines.foldLeft(Map.empty[String, Vector[String]].withDefault(_ => Vector.empty)) {
-      case (m, "")                                                      => m
-      case (m, linePattern(field @ (Data | Event | Id | Retry), value)) => m.updated(field, m(field) :+ value)
-      case (m, field)                                                   => m.updated(field, m(field) :+ "")
+  private def parseFields(lines: Vector[String]) = {
```

Comment:
In the old version there was a `lastOption` call applied to this result (for all fields except for `Data`), so it's exactly equivalent.